doc/GUIDE.md

```diff
@@ -1683,87 +1683,13 @@ page](https://docs.haskellstack.org/en/stable/shell_autocompletion)
 
 ### Docker
 
-stack provides two built-in Docker integrations. The first way is to
-build your code inside a Docker image, which means:
+Stack is able to build your code inside a Docker image, which means:
 
 * even more reproducibility to your builds, since you and the rest of your team
   will always have the same system libraries
 * the Docker images ship with entire precompiled snapshots. That means you have
```

Comment:
Maybe better for a separate PR, but noticed that this is no longer accurate (the `fpco/stack-build` image does _not_ include a precompiled snapshot).